src/Microsoft.AspNetCore.Server.IISIntegration.nuspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.AspNetCore.Server.IISIntegration</id>
+    <version>$version$</version>
+    <description>ASP.NET Core components for working with the IIS AspNetCoreModule.</description>
+    <tags>aspnetcore,iis</tags>
+    <serviceable>true</serviceable>
+    <dependencies>
+      <group targetFramework=".NETFramework4.5.1">
+        <dependency id="Microsoft.AspNetCore.Hosting.Abstractions" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.Http" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.Http.Extensions" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.HttpOverrides" version="$version$" />
+        <dependency id="Microsoft.Extensions.Logging.Abstractions" version="$version$" />
+        <dependency id="Microsoft.Extensions.Options" version="$version$" />
+      </group>
+      <group targetFramework=".NETStandard1.3">
+        <dependency id="System.Security.Principal.Windows" version="4.0.0" />
+        <dependency id="Microsoft.AspNetCore.Hosting.Abstractions" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.Http" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.Http.Extensions" version="$version$" />
+        <dependency id="Microsoft.AspNetCore.HttpOverrides" version="$version$" />
+        <dependency id="Microsoft.Extensions.Logging.Abstractions" version="$version$" />
+        <dependency id="Microsoft.Extensions.Options" version="$version$" />
```

Comment:
I'd rather we put like a bogus version in (possibly even an invalid semver so `nuget pack` will fail), or do the thing that MSBuild itself did and just don't specify the version so that when we try to build packages and use them we realize immediately that we need to solve this problem. This could silently "work" and then cause weird failures at runtime when people aren't paying attention to NuGet dependency versions.